### PR TITLE
Added ability to use sandbox mode

### PR DESCRIPTION
**Pull request by keithslater**
Ability to use sandbox mode.

Example usage

    $IPP = new QuickBooks_IPP($dsn);
    // Enables sandbox mode
    $IPP->sandbox(true);

### CODE REVIEW

**Comment by aik099** [comment]
Unfortunately this won't have any effect if the `QuickBooks_IPP` is instantiated internally by the library itself (e.g. https://github.com/consolibyte/quickbooks-php/blob/8d838c67628dfb4c25e03d89e0e7e3c982d761af/QuickBooks/IPP/Federator.php#L661).

**Comment by keithslater** [comment]
Interesting, I didn't realize that happened. I will close this and hope for a better solution.

**Comment by aik099** [comment]
Please reopen back because there are 2 ways of using the library: with federator and without it. Issue is only when federator is used, but it isn't used in any of examples, so I guess it's old authentication approach.

Really need some input from library developers.

**Comment by keithslater** [comment]
That makes sense. I just recently started using the library so I'm not that familiar with it.

**Comment by whobutsb** [comment]
Any ideas on when this will get updated to the project?

**Comment by aik099** [comment]
Knowing that last release was half a year ago and even isn't displayed on Packagist I doubt it we see new release this year.

**Comment by aik099** [comment]
I've noticed that you've sent PR from `master` branch. This is not correct way to do this, because any further commits added to `master` branch will be added to this PR.

Instead you should have created new branch your repo fork and sent pull request from it.

File: QuickBooks/IPP.php [hunk 1]
```diff
@@ -961,6 +963,22 @@ public function version($version = null)
...
+
+	/**
+	 * Get or set the sandbox mode. If not set, will default false for production
+	 *
+	 * @param bool $sandbox		Set as true for sandbox mode or false for production
+	 * @return bool				Returns true if in sandbox mode. False for production mode
+	 */
+	public function sandbox($sandbox = null)
+	{
+		if ($sandbox)
```
Comment (aik099) [inline QuickBooks/IPP.php]: You should do `isset($sandbox)` instead because currently if you turn on sandbox mode it's impossible to turn it off.

Comment (keithslater) [inline QuickBooks/IPP.php]: I actually meant to do $sandbox = false instead of null.

**Comment by aik099** [comment]
I suppose this library don't have any unit tests where you can add new one to verify new behavior.

**Comment by consolibyte** [comment]
Support for this was added in.

**Comment by aik099** [comment]
Thanks.

**Pull request closed without merging**